mobify/platform-scaffold
# Fix some 'live-only' issues

Some assets weren't being referenced correctly when running from lambda. This PR fixes that and a few other minor issues. 

~~**JIRA**: (link to JIRA ticket)~~

## Changes
- Rewrite stylesheets so url references are remapped to static assets properly (fixes chevron arrow assets not showing on staging/production)
- The above increases our memory footprint enough that lambda runs out of memory consistently, so increase memory from 128MB to 256MB
- Run on lambda with NODE_ENV set to ‘production’
- Fix PLP header icon not working on staging/production
- Include our own version of jQuery, instead of downloading it every time

## How to test-drive this PR
- Run `npm run dev`
- Preview [local Merlin's Potions PDP](http://localhost:3000/eye-of-newt.html) - observe chevrons still appear
- Preview [local Merlin's Potions PLP](http://localhost:3000/potions.html) - observe potions image still appears
- Preview [staging Merlin's Potions PDP](https://amp-staging.merlinspotions.com/eye-of-newt.html)
- Preview [staging Merlin's Potions PLP](https://amp-staging.merlinspotions.com/potions.html)
